x-pack/plugins/observability_solution/apm/tsconfig.json

```diff
@@ -128,6 +128,7 @@
     "@kbn/router-utils",
     "@kbn/react-hooks",
     "@kbn/alerting-comparators",
+    "@kbn/observability-utils"
```

Comment:
Do we still need this?

Reply:
No, it has been removed in https://github.com/elastic/kibana/pull/196387/commits/8c1bd535fc21675bcc9e6f54dc534edbf2d46110

Reply:
Quick question, @cauemarcondes: the changes in this file are what prompted the Kibana-Core team to be added as reviewers, correct? I just want to confirm before I remove them.